plugins/styleguide/assets/javascripts/discourse/components/sections/atoms/05-input-fields.hbs

```diff
@@ -1,108 +1 @@
-  <PasswordField type="password" placeholder="Placeholder" />
-</StyleguideExample>
-
-<StyleguideExample @title="textarea">
-  <Textarea placeholder="Placeholder" />
-</StyleguideExample>
-
-<StyleguideExample @title="inline-form">
-  <div class="inline-form">
-    <TextField @placeholder="Placeholder" />
-    <DButton
-      @icon="search"
-      @translatedLabel="Submit"
-      type="submit"
-      class="btn-primary"
-    />
-  </div>
-</StyleguideExample>
-
-<StyleguideExample @title="inline-form with icon button">
-  <div class="inline-form">
-    <TextField @placeholder="Placeholder" />
-    <DButton @icon="search" type="submit" class="btn-primary" />
-  </div>
-</StyleguideExample>
-
-<StyleguideExample @title="full-width inline-form with single input">
-  <div class="inline-form full-width">
-    <TextField @placeholder="Placeholder" />
-  </div>
-</StyleguideExample>
-
-<StyleguideExample @title="full-width inline-form with input and icon button">
-  <div class="inline-form full-width">
-    <TextField @placeholder="Placeholder" />
-    <DButton @icon="search" type="submit" class="btn-primary" />
-  </div>
-</StyleguideExample>
-
-<StyleguideExample
-  @title="inline-form with <ComboBox>"
-  @initialValue={{get @dummy "options.0.name"}}
-  as |value|
->
-  <div class="inline-form">
-    <TextField @placeholder="Placeholder" />
-    <ComboBox
-      @content={{@dummy.options}}
-      @value={{value}}
-      @onChange={{fn (mut value)}}
-    />
-    <DButton
-      @icon="search"
-      @translatedLabel="Submit"
-      type="submit"
-      class="btn-primary"
-    />
-  </div>
-</StyleguideExample>
-
-<StyleguideExample @title="inline-form with <MultiSelect>">
-  <div class="inline-form">
-    <TextField />
-    <MultiSelect @content={{@dummy.options}} @onChange={{@dummyAction}} />
-    <DButton
-      @icon="search"
-      @translatedLabel="Submit"
-      type="submit"
-      class="btn-primary"
-    />
-  </div>
-</StyleguideExample>
-
-<StyleguideExample @title="inline-form with <MultiSelect> and label">
-  <div class="inline-form">
-    <label>Text:</label>
-    <TextField />
-    <MultiSelect @content={{@dummy.options}} @onChange={{@dummyAction}} />
-    <DButton
-      @icon="search"
-      @translatedLabel="Submit"
-      type="submit"
-      class="btn-primary"
-    />
-  </div>
-</StyleguideExample>
-
-<StyleguideExample @title="full-width inline-form with search type input">
-  <div class="inline-form full-width">
-    <Input placeholder="Search type input" @type="search" />
-  </div>
-</StyleguideExample>
-
-<StyleguideExample @title="<CategoryNotificationsButton> and regular button">
-  <div class="inline-form">
-    <CategoryNotificationsButton
-      @category={{get @dummy.categories 0}}
-      @value={{1}}
-      @onChange={{@dummyAction}}
-    />
-    <DButton @icon="reply" @type="submit" @translatedLabel="Button" />
-  </div>
-</StyleguideExample>
\ No newline at end of file
+{{! Removed for now }}
```

Comment:
What's the plan here? I guess we should either add something, or remove it entirely?

Reply:
I need to find the motivation to write it :p